build/configs/rtl8721csm/tc/defconfig

```diff
@@ -249,27 +252,24 @@ CONFIG_FLASH_PART_NAME="user,"
 #
 # Kernel Features
 #
-CONFIG_DISABLE_OS_API=y
-# CONFIG_DISABLE_POSIX_TIMERS is not set
-# CONFIG_DISABLE_PTHREAD is not set
-# CONFIG_DISABLE_SIGNALS is not set
-# CONFIG_DISABLE_MQUEUE is not set
-# CONFIG_DISABLE_ENVIRON is not set
+# CONFIG_DISABLE_OS_API is not set
 
 #
 # Clocks and Timers
 #
-CONFIG_USEC_PER_TICK=1000
+CONFIG_ARCH_HAVE_TICKLESS=y
+# CONFIG_SCHED_TICKLESS is not set
+CONFIG_USEC_PER_TICK=10000
 CONFIG_SYSTEM_TIME64=y
-# CONFIG_CLOCK_MONOTONIC is not set
+CONFIG_CLOCK_MONOTONIC=y
 # CONFIG_JULIAN_TIME is not set
-CONFIG_START_YEAR=2011
-CONFIG_START_MONTH=12
-CONFIG_START_DAY=6
-CONFIG_MAX_WDOGPARMS=2
-CONFIG_PREALLOC_WDOGS=4
-CONFIG_WDOG_INTRESERVE=0
-CONFIG_PREALLOC_TIMERS=2
+CONFIG_START_YEAR=2014
```

Comment:
What is the reason you set the base year, month and day with this value?

Reply:
I just updated defconfig based on working rdp configuration.
Have removed it now, i think this is not needed,